### ISSUE TITLE
Differences between objective functions

### ISSUE THREAD

**Issue by ha0x7c7** [NicolasHug/Surprise#59]
Hi
I have seen in some papers that for computing error, authors use this term `e_ij = R_ij - U_i.dot(V_j)` but in some other papers I see this term: ` `e_ij = U_i.dot(V_j) - R_ij`, these two variations causes different update rules for parameters, I was wondering does this effect model parameters at the end?

**Comment by NicolasHug** [comment]
Hi,

Both are wrong anyway, as in both case some errors can be negative which makes no sense. In most loss functions errors are either squared, or at least made positive by taking the abs. value.

If you have seen different expression for e_ij, it was probably in an algorithm doing gradient descent. In this case the expression you use will have an effect on the update rule but that's all.

As a side note, this is a bug tracker. Your question would probably more suited somewhere on reddit or stackexchange.
Nicolas
